grayscaler/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import {CloudFormationClient, DescribeStackResourceCommand} from '@aws-sdk/client-cloudformation';
+import {GetObjectCommand, PutObjectCommand, S3Client} from '@aws-sdk/client-s3';
+import {Handler} from 'aws-lambda';
+import * as Jimp from 'jimp';
+
+const getBucketName = async (stackName: string, logicalResourceId: string) => {
```

Comment:
Did you consider binding the bucket name deployment time by setting then as environment variables?

Reply:
Yep, good idea. It is now implemented.